core/src/sql/value/value.rs

```diff
@@ -2880,19 +2880,19 @@ mod tests {
 
 	#[test]
 	fn check_serialize() {
-		let enc: Vec<u8> = Value::None.try_into().unwrap();
+		let enc: Vec<u8> = Value::None.into();
 		assert_eq!(2, enc.len());
-		let enc: Vec<u8> = Value::Null.try_into().unwrap();
+		let enc: Vec<u8> = Value::Null.into();
 		assert_eq!(2, enc.len());
-		let enc: Vec<u8> = Value::Bool(true).try_into().unwrap();
+		let enc: Vec<u8> = Value::Bool(true).into();
 		assert_eq!(3, enc.len());
-		let enc: Vec<u8> = Value::Bool(false).try_into().unwrap();
+		let enc: Vec<u8> = Value::Bool(false).into();
 		assert_eq!(3, enc.len());
-		let enc: Vec<u8> = Value::from("test").try_into().unwrap();
+		let enc: Vec<u8> = Value::from("test").into();
 		assert_eq!(8, enc.len());
-		let enc: Vec<u8> = Value::parse("{ hello: 'world' }").try_into().unwrap();
+		let enc: Vec<u8> = Value::parse("{ hello: 'world' }").into();
 		assert_eq!(19, enc.len());
-		let enc: Vec<u8> = Value::parse("{ compact: true, schema: 0 }").try_into().unwrap();
+		let enc: Vec<u8> = Value::parse("{ compact: true, schema: 0 }").into();
```

Comment:
This code was altered indeed by `clippy --fix`. I don't really like the change either, and I think value should not implement Into<Vec<u8>>. But because it does clippy generates warnings.

Reply:
I can undo the changes and add a `#[allow(clippy::warning)]`?

Reply:
No no, let's keep the change, but we do need to move to using `try_into()` everywhere sometime soon!